ansible/vars/common.yaml

```diff
@@ -32,5 +32,5 @@ use_https: "yes"
 pypi_repo: "https://nexus-local.cc.epm.kharlamov.biz/repository/pypi-hosted/simple"
 docker_repo: "nexus-local.cc.epm.kharlamov.biz:443/legion"
 enclaves:
```

Comment:
Are we still using "enclave" var anywhere?

Reply:
Yep. I don't want to rewrite all ansible roles because we have a plan to get rid of them.